Doc/howto/regex.rst

```diff
@@ -449,6 +453,16 @@ case. ::
    >>> m.span()
    (4, 11)
 
+The :meth:`~re.Pattern.fullmatch` method checks if the RE matches the entire
+string exactly::
+
+   >>> p = re.compile('[a-z]+')
+   >>> p.search(' words ')
+   <re.Match object; span=(1, 6), match='words'>
```

Comment:
Use the same filler text throughout the example snippet?
```suggestion
   >>> p = re.compile('[a-z]+')
   >>> p.search(' textual ')
   <re.Match object; span=(1, 6), match='textual'>
```

Reply:
Done!